infra/cifuzz/cifuzz_test.py

```diff
@@ -114,60 +122,53 @@ def test_invalid_workspace(self):
 class RunFuzzersIntegrationTest(unittest.TestCase):
   """Test build_fuzzers function in the cifuzz module."""
 
+  def tearDown(self):
+    """Remove any existing crashes and test files."""
```

Comment:
why do we need to delete all of those? maybe we can be more concrete and delete only when needed, instead of deleting everything that is not fuzz target binary?

Reply:
I don't want to corrupt the sanity of the other tests.